plugin-server/package.json

```diff
@@ -89,7 +89,7 @@
         "node-schedule": "^2.1.0",
         "pg": "^8.6.0",
         "pino": "^8.6.0",
-        "posthog-node": "2.0.2",
+        "posthog-node": "4.5.0",
```

Comment:
That's a serious amount of version bumps for our own product 😅

Reply:
Actually, let's get https://github.com/PostHog/posthog-js-lite/pull/390 out first and upgrade to that in a separate PR first. I don't want to conflate a version upgrade with anything error tracking specific